src/panels/lovelace/cards/hui-media-control-card.ts

```diff
@@ -218,23 +305,36 @@ export class HuiMediaControlCard extends LitElement implements LovelaceCard {
     const newImage = this.hass.states[this._config.entity]?.attributes
       .entity_picture;
 
-    if (!newImage || newImage === oldImage) {
+    if (!newImage) {
       this._image = newImage;
```

Comment:
Is there any reason we would store this as an instance variable instead of just having a getter ?

Reply:
Something like this?
```js
  private get _image() {
    if (!this.hass || !this._config) {
      return "../static/images/card_media_player_bg.png";
    }

    return (
      this.hass.states[this._config.entity]?.attributes.entity_picture_local ||
      this.hass.states[this._config.entity]?.attributes.entity_picture ||
      "../static/images/card_media_player_bg.png"
    );
  }
```

Reply:
If its updated though we want to re-render

Reply:
don't we already rerender if hass or config is changed?

Reply:
Yep
Forgot to update this comment. That it rerendered because of that